liferay-tango/liferay-portal
# LPS-119068 [Page Audit] Performance Issue

**Motivation**

LPS-119068 introduces about 5% regression to portal because the LayoutReportsPortlet gets rendered even if the panel is not expanded.

Please see the comparison below:

Git id | Session Count | Diff
-- | -- | --
e3019ac7 | 13885 |  
e3019ac8 (Disable PageSpeed) | 14578 | +4.990997479


**Proposed solution**

Avoid using RuntimeTag to dispatch the Portlet, use React Renderer instead.

**How to test it**

1. Go to a Content/Widget/DPT Page
2. Click the info button
3. The Page Audit graph should be visible

**_We should wait until performance test are executed in this PR to know if this is a valid solution._**
